recipes/intel-ipp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "intel-ipp" %}
+{% set version = "2019.1.144" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/14887/l_ipp_{{ version }}.tgz
+  sha256: 1eb7cd0fba74615aeafa4e314c645414497eb73f1705200c524fe78f00620db3
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Seems like you actually need python here, not a c compiler

Reply:
Good catch, thanks